sw/ground_segment/cockpit/live.ml

```diff
@@ -1521,6 +1532,45 @@ let get_shapes = fun (geomap:G.widget)_sender vs ->
 let listen_shapes = fun (geomap:G.widget) ->
   safe_bind "SHAPE" (get_shapes geomap)
 
+let get_cam_status = fun (geomap:MapCanvas.widget) _sender vs ->
```

Comment:
The Horizontal field of view and the vertical field of view of the camera may not be the same. We need one program to read the fields from airframe.xml. If the camera is pointed straight down the target point will be directly under the aircraft no matter what angle the camera is panned at. Due to pan angle the footprint may be a wide rectangle, a skinny rectangle, or a diamond shape. This test case can then be applied to a variety of tilt angle. Am I missing something with the target point? The pieces of information we need to fully calculate a footprint are ac.pitch, ac.roll, ac.yaw, ac.agl, ac.pos, cam.pan, cam.tilt, cam.hfv, and cam.vhf. IMHO the server should send cam.pan and cam.tilt messages to the GCS, GCS will then compute the footprint (having all the information), GCS will then send the the information to mapTrack.